model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -77,7 +77,13 @@ class LicenseInfoResolver(
 
         // Handle concluded licenses.
         concludedLicenses.forEach { license ->
-            license.builder().sources += LicenseSource.CONCLUDED
+            license.builder().apply {
+                sources += LicenseSource.CONCLUDED
+
+                licenseInfo.concludedLicenseInfo.concludedLicense?.let {
+                    originalExpressions.add(it)
```

Comment:
Can we use `+=` instead of `.add()` here?

Reply:
Same as [here](https://github.com/oss-review-toolkit/ort/pull/3616#discussion_r574309705)